components/Shell.tsx

```diff
@@ -37,15 +37,8 @@ import { useViewerI18n } from "./I18nLanguageHandler";
 import Logo from "./Logo";
 
 function useMeQuery() {
-  const [session] = useSession();
   const meQuery = trpc.useQuery(["viewer.me"]);
 
-  useEffect(() => {
-    // refetch if sesion changes
-    meQuery.refetch();
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [session]);
```

Comment:
Thought this was a good idea but it a) doesn't seem to matter and b) refetches several times as the `session` seems unstable